packages/cross_file/lib/src/types/html.dart

```diff
@@ -64,12 +64,8 @@ class XFile extends XFileBase {
         _lastModified = lastModified ?? DateTime.fromMillisecondsSinceEpoch(0),
         _name = name ?? '',
         super(path) {
-    if (path == null) {
-      _browserBlob = _createBlobFromBytes(bytes, mimeType);
-      _path = URL.createObjectURL(_browserBlob!);
-    } else {
-      _path = path;
-    }
+    _browserBlob = _createBlobFromBytes(bytes, mimeType);
```

Comment:
The unnamed constructor always sets the blob when the bytes are provided.
This one didn't but we can always do that since the bytes are _required_ for this one.